python/ray/serve/_private/proxy.py

```diff
@@ -742,7 +743,7 @@ async def send_request_to_replica(
 
             yield ResponseStatus(code=grpc.StatusCode.OK)
         except TimeoutError:
-            message = f"Request {request_id} timed out after {self.request_timeout_s}s."
+            message = f"Request timed out after {self.request_timeout_s}s."
```

Comment:
Why remove the request id? Same question for all the new log statements.

I find it useful to include the request id, so we can trace a request through the different actors.

Reply:
The request_id is already present in the logs in the structured metadata